spec/services/business_time_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+# rubocop:disable RSpec/RepeatedExample
+RSpec.describe BusinessTime do
+  [
+    Date.new(2019, 11, 4),
+    Date.new(2019, 11, 5),
+    Date.new(2019, 11, 6),
+    Date.new(2019, 11, 7),
+    Date.new(2019, 11, 8),
+  ].each do |date|
```

Comment:
Not sure about this.

- I find this spec hard to follow as I don't know (without looking it up) that these are all weekdays
- I feel ambivalent about testing this as it's `BusinessTime`'s job to handle it? (For the intersection of `BusinessTime` and `Holiday` I see a bit of value)

Reply:
I'm not too sure about these either. I can perhaps update the descriptions to make them a little clear. But whether we should include these at all is still an open question as far as I'm concerned. I'm thinking that if we have other higher level feature specs that exercise the working days logic and config then we don't need these. Drop them?

Reply:
Removed as discussed.